src/main/java/org/irods/operations/UserGroupOperations.java

```diff
@@ -55,6 +91,16 @@ public Response setPassword(String token, String name, String zone, String newPa
         return new Response(response.statusCode(), response.body());
     }
 
+    /**
+     * Changes a user's type. This operation requires rodsadmin level privileges.
+     *
+     * @param token The authentication token to use for the request.
+     * @param name Name of user.
+     * @param zone Name of zone for user.
+     * @param newUserType rodsuser, groupadmin, or rodsadmin. The new user type to be set to. Takes in the
```

Comment:
is this the right/normal way to enumerate the possible values?   feels more expected to have them at the end?

Reply:
I had it this way because for the flags, like `admin`, I would put 0 or 1 at the beginning. Not sure if it's the normal way but I could definitely move it to the end.

Reply:
Yes please. Describe the parameter first. If all values of `UserType` are acceptable, then you do not need to include them in the docs because issues will be caught at compile-time.

Reply:
Ok sounds good, I'll remove it.